src/applications/vaos/new-appointment/components/ReviewPage/ContactDetailSection.jsx

```diff
@@ -72,12 +84,12 @@ export default function ContactDetailSection({ data }) {
             </span>
           </div>
           <div>
-            <Link
-              to={newAppointmentFlow.contactInfo.url}
+            <VaLink
```

Comment:
Let's use the `va-link` for consistency
```suggestion
            <va-link
```